DigitalLearningSolutions.Data/DataServices/UserDataService/AdminUserDataService.cs

```diff
@@ -132,7 +132,8 @@ ELSE cc.CategoryName
                         au.IsWorkforceContributor,
                         au.IsLocalWorkforceManager,
                         au.ImportOnly,
-                        au.FailedLoginCount
+                        au.FailedLoginCount,
+                        au.ResetPasswordId

```

Comment:
On further investigation it turned out the lookup also doesn't work for admins. The actual reset login works fine because it uses its own methods, but I don't think the old password resets were ever getting deleted. It evaded our unit testing because the result is a formally valid output from the data service, and the service-service could assume useful data.